README.md

```diff
@@ -36,9 +36,10 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
 * We don’t handle support via e-mail, Twitter, GitHub issues etc.
 
 ### Contribute ###
-* Active development of this plugin is handled on GitHub.
+* Active development of this plugin is handled on GitHub. (see here: https://github.com/pluginkollektiv/cachify)
```

Comment:
As this is Markdown, I'd rather use links à la `[…] handled [on GitHub](https://github.com/pluginkollektiv/cachify).`